### PR TITLE
Proofpoint observe it

### PR DESCRIPTION
**Pull request by yasta5**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-4663

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

File: Packs/ProofpointObserveIT/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Proofpoint ObserveIT
+This pack includes Cortex XSIAM content. 
+
+## Configuration on Server Side
+You need to configure Proofpoint ObserveIT CEF log integration.
+
+Follow the below steps to configure CEF log integration:
+1. Go to **Configuration** &rarr;  **Integrations** &rarr; **Integrated SIEM**
```
Comment (ShirleyDenkberg) [inline Packs/ProofpointObserveIT/README.md:8]:
```suggestion
1. Go to **Configuration** > **Integrations** > **Integrated SIEM**.
```

File: Packs/ProofpointObserveIT/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# Proofpoint ObserveIT
+This pack includes Cortex XSIAM content. 
+
+## Configuration on Server Side
+You need to configure Proofpoint ObserveIT CEF log integration.
+
+Follow the below steps to configure CEF log integration:
+1. Go to **Configuration** &rarr;  **Integrations** &rarr; **Integrated SIEM** 
+2. Click on the **SIEM Log Integration** tab.
```
Comment (ShirleyDenkberg) [inline Packs/ProofpointObserveIT/README.md:9]:
```suggestion
2. Click the **SIEM Log Integration** tab.
```

File: Packs/ProofpointObserveIT/README.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+3. Select the **Enable export to ARCSight format** check box.
```
Comment (ShirleyDenkberg) [inline Packs/ProofpointObserveIT/README.md:10]:
```suggestion
3. Select the **Enable export to ARCSight format** checkbox.
```

File: Packs/ProofpointObserveIT/README.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+4. In the **Log data** section select **Activity Alerts** (should be selected by default).
```
Comment (ShirleyDenkberg) [inline Packs/ProofpointObserveIT/README.md:11]:
```suggestion
4. In the **Log data** section, select **Activity Alerts** (should be selected by default).
```

File: Packs/ProofpointObserveIT/README.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+5. In the **Log file properties** section, accept the default log file location and name:
+   - Path: C:\Program File\ObserveIT\NotificationService\LogFiles\ArcSight
+   - Name: Observeit_activity_log.cef
+6. In the **Log file cleanup** section schedule the frequency for clearing the log file.
```
Comment (ShirleyDenkberg) [inline Packs/ProofpointObserveIT/README.md:15]:
```suggestion
6. In the **Log file cleanup** section, schedule the frequency for clearing the log file.
```

File: Packs/ProofpointObserveIT/README.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+7. Click **Save**.
+
+More information can be found [here](https://documentation.observeit.com/configuration_guide/configuring_cef_log_integratio)
```
Comment (ShirleyDenkberg) [inline Packs/ProofpointObserveIT/README.md:18]:
```suggestion
More information can be found [here](https://documentation.observeit.com/configuration_guide/configuring_cef_log_integration.htm)
```

**Comment by ShirleyDenkberg** [comment]
@evisochek Doc review completed.

**Review by evisochek — APPROVED** [review]

**Pull request merged**